main_test.go

```diff
@@ -57,3 +58,141 @@ func TestNormalizeAddr(t *testing.T) {
 		})
 	}
 }
+
+func TestParseConfigDefaults(t *testing.T) {
+	ensureEnvVarsAreUnset()
```

Comment:
Would `os.Clearenv()` work instead?

Reply:
At first when I read this comment, I thought, yes, that would be so much more succinct. However, as I think about it, I worry about inadvertently clearing other env variables unrelated to this test. If the test were run in a developer's local command-line environment, or on a remote build server, it might have negative side-effects if it cleared certain variables. 

In fact, to be more responsible about it, I should probably store the original values for those variables and reset them at the end... I will add code to that effect.

Reply:
Just pushed this up.

Reply:
Any env var changes will only affect that process (i.e. the running test), and not the parent process invoking the test (CLI, for example). I double checked this by setting some of the used env vars in my shell and removing the restore call, and they were all fine after the test.

Reply:
Oh interesting! I wouldn't have guessed that. That's awesome! OK, I updated and pushed the code.